### ISSUE TITLE
The configration about CNC and laser.

### ISSUE THREAD

**Issue by FuqiangSong** [repetier/Repetier-Firmware#888]
Hello all,
Do you know the Repetier-Fiemware support CNC and Laser?
If so , and how to change the configration file?

**Comment by repetier** [comment]
Yes in a simple way. Just enable/disable no power control. If you use the config tool you can activate the options. Then you just switch mode
- M450 - Reports printer mode
- M451 - Set printer mode to FFF
- M452 - Set printer mode to laser
- M453 - Set printer mode to CNC

and the extruder will behave as laser or cnc with configured pin. If you have no real extruder use fake temperature sensor and -1 as heater pin.
